tests/main/degraded/task.yaml

```diff
@@ -13,7 +13,7 @@ debug: |
         units="$(systemctl --failed --type=$unit_type --no-pager | grep -o -E ".*.$unit_type" | tr '●' ' ' )"
         for unit in $units; do
             echo " -- systemctl status $unit --"
-            systemctl status "$unit"
+            systemctl status "$unit" || true
```

Comment:
What is the motivation behind this change? When would status fail?

Reply:
It's in the commit message. To recap, `systemctl status <svc>` exits with non-0 status when the service has failed.

Reply:
Interesting, thanks.